index.js

```diff
@@ -223,6 +224,7 @@ function fromJsonObject (that, object) {
 
 if (Buffer.TYPED_ARRAY_SUPPORT) {
   Buffer.prototype.__proto__ = Uint8Array.prototype
+  Buffer.prototype._set = Uint8Array.prototype.set // For `copy` below.
```

Comment:
I know you just copied this but I wonder if we can't just use `Uint8Array.prototype.set.call` instead of saving a reference here.